discord/guild.py

```diff
@@ -1404,6 +1405,53 @@ def fetch_members(self, *, limit=1, after=None):
             result.append(Invite(state=self._state, data=invite))
 
         return result
+      
+    async def create_integration(self, integration_type, integration_id):
```

Comment:
```suggestion
    async def create_integration(self, *, type, id):
```

Library tends to use kwarg-only args for things like this.